benches/blocking.rs

```diff
@@ -0,0 +1,26 @@
+#![feature(test)]
+
+extern crate test;
+
+use async_std::task;
+use async_std::task::blocking::JoinHandle;
+use futures::future::join_all;
+use std::thread;
+use std::time::Duration;
+use test::Bencher;
+
+#[bench]
+fn blocking(b: &mut Bencher) {
+    b.iter(|| {
+        let handles = (0..10_000)
```

Comment:
That's great, how about swapning >MAX_THREADS with thread::sleep(10sec)? Will your patch change the behavior?